src/libs/actions/TaxRate.ts

```diff
@@ -0,0 +1,129 @@
+import Onyx from 'react-native-onyx';
+import * as API from '@libs/API';
+import type {CreatePolicyTaxParams} from '@libs/API/parameters';
+import {WRITE_COMMANDS} from '@libs/API/types';
+import CONST from '@src/CONST';
+import * as ErrorUtils from '@src/libs/ErrorUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {TaxRate, TaxRates} from '@src/types/onyx';
+import type {PendingAction} from '@src/types/onyx/OnyxCommon';
+import type {OnyxData} from '@src/types/onyx/Request';
+
+/**
+ * Get tax value with percentage
+ */
+function getTaxValueWithPercentage(value: string): string {
+    return `${value}%`;
```

Comment:
I believe using a normal concat is more performant than this, I remember we made such change in the past cc @gedu

Reply:
I can change it, but it's due to a lint error:

Unexpected string concatenation.eslint[prefer-template](https://eslint.org/docs/rules/prefer-template)

Should I ignore it?

Reply:
We can keep that for future, no need to change now